packages/fxa-settings/src/lib/hooks.tsx

```diff
@@ -50,3 +62,31 @@ export function useChangeFocusEffect() {
 
   return elToFocus;
 }
+
+// TODO - Iron out how we want to handle mutation errors (FXA-2450)
+// We must have an `onError` option in mutations to allow tests to pass
+// but providing one prevents an error from actually throwing so we need
+// to manually report it to Sentry.
+// See https://github.com/apollographql/react-apollo/issues/2614
+export function useHandledMutation(
+  mutation: DocumentNode,
+  options: MutationHookOptions<any, Record<string, any>> | undefined = {}
+) {
+  return useMutation(
+    mutation,
+    Object.assign(options, {
+      // Pass in options.onError to handle an error and
+      // optionally re-throw so it gets logged to Sentry
+      // By default errors are ignored
+      onError: (error: ApolloError) => {
+        if (options.onError) {
```

Comment:
idk that ignore by default is what we want, maybe my prev comment was misleading... If no `onError` handler is passed I think we _do_ want to send to sentry & maybe console because in that case I think not passing in a handler is like, "I don't know of any errors that can happen here, lmk if there are".

Reply:
i can dig it